components/oauth/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/GrantType.java

```diff
@@ -20,7 +20,9 @@
 public enum GrantType {
 
     SAML20_BEARER("urn:ietf:params:oauth:grant-type:saml2-bearer"),
-    IWA_NTLM("iwa:ntlm");
+    IWA_NTLM("iwa:ntlm"),
+    JWT("urn:ietf:params:oauth:grant-type:jwt-bearer");
```

Comment:
Please do not change anything here. Make the grant type self-contained so it can be shipped as an extension without patching.